packages/node_modules/@ciscospark/internal-plugin-metrics/test/unit/spec/metrics.js

```diff
@@ -31,7 +31,8 @@ describe(`plugin-metrics`, () => {
         testTag: `tag value`
       },
       metricName: eventName,
-      type: `behavioral`
+      type: `behavioral`,
+      timestamp: Date.now()
```

Comment:
This could be flaky if the mock and the src code happen to call Date.now() on a transition from one millisecond to the next. You should be able to use lolex to set a particular "now" so that the test is deterministic.